src/qibolab/instruments/zhinst.py

```diff
@@ -55,8 +55,8 @@
 SWEEPER_BIAS = {"bias"}
 SWEEPER_START = {"start"}
 
-MAX_MEASUREMENTS = 32
-"""Maximum number of readout pulses in a single sequence."""
+MAX_SEQUENCES = 150
```

Comment:
I have not done extensive benchmarking about this, but as a general note it is better to underestimate than push it to its limit, because execution will fail completely if we exceed the memory. While if we underestimate, it may take a bit longer because it will split more batches but it will still work.

Reply:
Yeah I agree, I found that for RB like circuits it would be like 20 while for simple calibration experiments it was around 150. So it is a huge change, I would suggest to count the number of pulses in the sequence as part of the batching maybe.